.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 rvm:
-- 2.4.2
+- 2.4.3
```

Comment:
I don't think we need to specify this at all. Travis will read the `.ruby-version` https://docs.travis-ci.com/user/languages/ruby/#Using-.ruby-version